builtin/logical/ad/backend.go

```diff
@@ -1 +1,25 @@
 package ad
+
+import (
+	"context"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func Factory(_ context.Context, conf *logical.BackendConfig) (logical.Backend, error) {
+
```

Comment:
We might want to call framework.backend.Setup() here to set the logger and the system view in the Backend object.

Reply:
Good catch! Will do!